src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -142,6 +143,7 @@ const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
     <ReactionsProvider venueId={venue.id}>
       <WithNavigationBar fullscreen={fullscreen} hasBackButton={hasBackButton}>
         <AnnouncementMessage message={venue.bannerMessage} />
+        <WithInformationCard venue={venue} showInfo={venue?.showInfo} />
```

Comment:
As mentioned in an earlier comment, it might be better to conditionally render this here directly rather than passing the `showInfo` prop into the component. Also, `venue` is always defined at this point, so there is no reason to use the `?` 'safe accessor' helper:

```suggestion
        {venue.showInfo && <WithInformationCard venue={venue} />}
```

Reply:
Also, how will this 'interact' with other templates throughout the app that already explicitly use `InformationLeftColumn`/related wrappers/etc themselves directly? I expect by rendering it here _and_ within those components, the floating elements are going to end up 'over the top of one another':

- https://github.com/sparkletown/sparkle/search?q=InformationLeftColumn
- https://github.com/sparkletown/sparkle/search?q=InformationCard

eg.

- `DonatePopUp`: https://github.com/sparkletown/sparkle/blob/staging/src/components/molecules/DonatePopUp/DonatePopUp.tsx#L24-L48
  - seemingly only used on `src/components/templates/Playa/Playa.tsx` (legacy template)
- `SparkleFairiesPopUp`: https://github.com/sparkletown/sparkle/blob/staging/src/components/molecules/SparkleFairiesPopUp/SparkleFairiesPopUp.tsx#L17-L46
  - used on
    - `src/components/templates/PartyMap/PartyMap.tsx`
    - `src/components/templates/ArtPiece/ArtPiece.tsx`
    - `src/components/templates/Playa/Playa.tsx` (legacy template)
- `CreateEditPopUp`: https://github.com/sparkletown/sparkle/blob/staging/src/components/molecules/CreateEditPopUp/CreateEditPopUp.tsx
  - seemingly only used on `src/components/templates/Playa/Playa.tsx` (legacy template)
- `JazzBarSkeletonPage`: https://github.com/sparkletown/sparkle/blob/staging/src/components/templates/Jazzbar/JazzBarSkeletonPage/JazzBarSkeletonPage.tsx#L23-L77
  - used on `src/components/templates/Jazzbar/Jazzbar.tsx`
  - the usage in `JazzBarSkeletonPage` seems to be the same venue name/subtitle/description usage as implemented in this PR + some default fallbacks to legacy text (Kansas Smitty's). I expect we could probably clean up all of this and just use your implementation here.
    - In doing so, it may be possible to remove the `JazzBarSkeletonPage` component entirely, and just move any relevant styles/etc (eg. `jazz-bar-container`) into `Jazzbar` directly.
    - And then once that is done, it makes `src/components/templates/Jazzbar/Jazzbar.tsx` sort of a basically empty/irrelevant wrapper too, so it could probably be removed, and rename `src/components/templates/Jazzbar/JazzTab/JazzTab.tsx` to `Jazzbar` (+ move it up a level so all of it's files end up in `src/components/templates/Jazzbar/` rather than being in `src/components/templates/Jazzbar/JazzTab`)
- `ArtPiece`: https://github.com/sparkletown/sparkle/blob/staging/src/components/templates/ArtPiece/ArtPiece.tsx#L32-L42
  - used in `src/pages/VenuePage/TemplateWrapper.tsx`
  - it also looks like the `default` exports aren't actually used anywhere, so can be removed from `ArtPiece.tsx` + the associated `index.tsx`
- `ConversationSpace`: https://github.com/sparkletown/sparkle/blob/staging/src/components/templates/ConversationSpace/ConversationSpace.tsx#L36-L46
  - used in `src/pages/VenuePage/TemplateWrapper.tsx`
  - it also looks like the `default` exports aren't actually used anywhere, so can be removed from `ConversationSpace.tsx` + the associated `index.tsx`
- etc